wod_predictor/modeling.py

```diff
@@ -54,7 +46,29 @@ def show_results(self, y_test, meta_data={}):
 
             show_breakdown_by_workout(y_pred_unstacked, y_test_unstacked)
 
+        self.get_comparative_stats()
+
+    # get comparative stats
+    def get_comparative_stats(self):
+        baseline_model = BaselineModel()
```

Comment:
Just suggesting an additional feature: can we allow users to use an arbitrary model (as long as it inherits from BaseModeler) as the baseline model?

Reply:
Thats actually a good idea, will skip implementing it for now but adding to the backlog.